oemoflex/model/postprocessing.py

```diff
@@ -53,6 +54,24 @@ def get_scalars(dict):
     return scalars
 
 
+def drop_component_to_component(series):
+    r"""
+    Drops those entries of an oemof_tuple indexed Series
+    where both target and source are components.
+    """
+    _series = series.copy()
```

Comment:
Why are you copying series when you want to drop related rows anyway?

Reply:
You are right, this is unnecessary for the way the function is used currently. Copying is only necessary if it might be used in a context where one wants to keep the unaltered data. In case it does not copy, does it make sense to add a line to the docstring that warns against that?

Reply:
hmm you are right - as oemoflex is supposed to be used by other projects, we should only modify the copy (in oemof_b3 module, I would have skipped copying). Sorry for confusion.